warehouse/models/mart/transit_database/_mart_transit_database.yml

```diff
@@ -9,10 +9,26 @@ models:
         tests: &primary_key_tests
           - not_null
           - unique
+        meta:
+          metabase.semantic_type: type/PK
```

Comment:
I wonder if we should alter dbt-metabase to flag PKs based on not-null and unique tests existing?

Reply:
nope